### ISSUE TITLE
`cbt --help` outputs unevaluated variables

### ISSUE THREAD

**Issue by mbrukman** [GoogleCloudPlatform/google-cloud-go#852]
If we run:

```
cbt --help 2>&1 | tail -1
```

we get the following output:

```
cbt ` + version + ` ` + revision + ` ` + revisionDate + `
```

Presumably, we should be output the *values* of those variables, not their names. Looks like this is due to
[this line](https://github.com/GoogleCloudPlatform/google-cloud-go/blob/48539605f0fd577f22ef648b929c96045e803a2e/bigtable/cmd/cbt/cbt.go#L176), which says:

```go
buf.WriteString("\ncbt ` + version + ` ` + revision + ` ` + revisionDate + `")
```

A few additional minor points:
* that line should also have a newline at the end
* presumably, we should use a platform-independent way of outputting newlines, such that we get `\n` on *nix and `\r\n` on Windows

**Comment by garye** [comment]
FYI this only happens when you build `cbt` locally, not when you use the `gcloud` version or anything that went through an official release process.

**Comment by enocom** [comment]
This doesn't seem to be a problem anymore. When I build locally against 5f0ffe7729373c98a5801de148565a893a0d2899 and run:

```
./cbt --help 2>&1 | tail -1
```

I get:

```
cbt <unknown version> <unknown revision> <unknown revision date>
```

Further, if I run the same command using cbt as included in gcloud v220.0.0, I get:

```
cbt 0.6.1 290bef81 2018-08-21 21:52:26
```

@mbrukman Feel free to re-open this issue if I've missed something.
